prefetch.bs

```diff
@@ -290,12 +296,12 @@ The user agent may [=prefetch record/cancel and discard=] records from the [=Doc
     1. Let |recordToUse| be |exactRecord| if |exactRecord| is not null, otherwise |inexactRecord|.
     1. If |recordToUse| is not null:
         1. Let |currentTime| be the [=current high resolution time=] for |navigable|'s [=navigable/active document=].
-        1. If |recordToUse|'s [=prefetch record/expiry time=] is less than |currentTime|, return null.
+        1. If |recordToUse|'s [=prefetch record/expiry time=] is less than |currentTime|, return null and [=trigger a prefetch status updated event=] given |navigable|, |recordToUse|, and "{^speculation.PreloadingStatus/failure^}" status
         1. [=list/For each=] |exchangeRecord| of |recordToUse|'s [=prefetch record/redirect chain=]:
-            1. If [=conflicting credentials exist=] for |exchangeRecord|'s [=exchange record/response=] given |navigable| and |recordToUse|'s [=prefetch record/source partition key=], return null.
+            1. If [=conflicting credentials exist=] for |exchangeRecord|'s [=exchange record/response=] given |navigable| and |recordToUse|'s [=prefetch record/source partition key=], return null and [=trigger a prefetch status updated event=] given |navigable|, |recordToUse|, and "{^speculation.PreloadingStatus/failure^}" status
 
             <div class="note">This handles the case where there were no cross-partition credentials initially, but there are now. User agents could use a slightly coarser algorithm, such as monitoring whether the cookies for the URL have been modified at all, or storing a hash, timestamp or revision number.</div>
-
+        1. [=Trigger a prefetch status updated event=] given |navigable|, |recordToUse|, and "{^speculation.PreloadingStatus/success^}" status.
```

Comment:
I suspect this is too early for success. Probably more around https://storage.googleapis.com/spec-previews/WICG/nav-speculation/pull/403/head/prefetch.html#ref-for-wait-for-a-matching-prefetch-record%E2%91%A0:~:text=Set%20prefetched%20to%20true. ?